flutter/Graminsta/lib/createPostWidget.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+
+class CreatePostWidget extends StatefulWidget {
+  CreatePostWidget({Key key}) : super(key: key);
+
+  @override
+  _CreatePostWidgetState createState() => _CreatePostWidgetState();
+}
+
+class _CreatePostWidgetState extends State<CreatePostWidget> {
+  String accessSetting = "All";
+
+  @override
+  Widget build(BuildContext context) {
+    double width = MediaQuery.of(context).size.width;
+
+    double height = MediaQuery.of(context).size.height;
+
+    return new Scaffold(
+        appBar: AppBar(title: Text('Create a Post'), actions: <Widget>[
+          IconButton(
+            icon: Icon(
+              Icons.save,
+              color: Colors.white,
+            ),
+            onPressed: () {
+              // do something
+            },
+          )
+        ]),
```

Comment:
Add trailing commas for all function calls and definitions, ditto for all.

Exception: function call or definition, which can be fitted in one line.

Reply:
One line: 80 chars.

Reply:
@shiyi001 @hht-hannah @ArianaGu FYI

Reply:
> One line: 80 chars.

I used flutter format to form the file, does it exceeds the limit?

Reply:
As per https://dart.dev/guides/language/effective-dart/style#avoid-lines-longer-than-80-characters, you should have at most 80 chars per line.

@shiyi001 @hht-hannah @ArianaGu FYI

Reply:
I set a ruler here and it seems not exceeding the 80 chars limit.
![ruler](https://user-images.githubusercontent.com/41231856/89881481-26f23000-dbf8-11ea-90de-da8552cc516d.png)